### PR TITLE
SSL Fix and compatibility issues.

### PR DESCRIPTION
**Pull request by isht3**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sudo: false
 python:
```
Comment (RishabhJain2018) [inline .travis.yml]: Please add it in a new PR.

File: setup.py [hunk 2]
```diff
@@ -39,6 +39,7 @@
         'python-dateutil==2.7.3',
```
Comment (RishabhJain2018) [inline setup.py]: Please add `travis` and `setup.py` changes in a new PR.

**Comment by isht3** [comment]
@RishabhJain2018 Done!

**Comment by isht3** [comment]
@RishabhJain2018 Understood. Closing this and opening a new one.

**Pull request closed without merging**